src/application/health_checks/health_checks.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * health_checks.h
+ *
+ * Created on: Nov 24, 2024
+ *
+ */
+
+#ifndef HEALTH_CHECKS_H_
+#define HEALTH_CHECKS_H
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#define ADC1_MAX_COUNTS 65535.0
+#define ADC1_VREF 3.3
+#define R_SENSE 0.033
+#define INA180A3_GAIN 100.0
+
+#define ADC1_VOLTAGE_V(counts) (counts / ADC1_MAX_COUNTS * ADC1_VREF)
```

Comment:
`#define ADC1_VOLTAGE_V(counts) (((float)(counts)) / ADC1_MAX_COUNTS * ADC1_VREF)`

Otherwise you may cause integer division errors